examples/ars.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+// Ars Technica, Open Source tag example
+
+const Libingester = require('libingester');
+const thenifyAll = require('thenify-all');
+const RssToJson = thenifyAll(require('rss-to-json'), {}, ['load']);
+const fs = thenifyAll(require('fs'), {}, ['writeFile']);
```

Comment:
I'm really against this pattern as `fs` generally has non-blocking methods but we use it in a lot things in the same manner so not a big deal for now.